src/jitlayers.cpp

```diff
@@ -129,12 +130,16 @@ void addOptimizationPasses(PassManager *PM)
     if (jl_options.opt_level == 0) {
         PM->add(createCFGSimplificationPass()); // Clean up disgusting code
         PM->add(createMemCpyOptPass()); // Remove memcpy / form memset
-        PM->add(createLowerPTLSPass(imaging_mode));
 #if JL_LLVM_VERSION >= 40000
         PM->add(createAlwaysInlinerLegacyPass()); // Respect always_inline
 #else
         PM->add(createAlwaysInlinerPass()); // Respect always_inline
 #endif
```

Comment:
are we actually confident that this can't break your gc-invariants and end up trying to run that pass on external code that it wasn't supposed to try to run on?